subprojects/core/src/main/java/org/gradle/api/internal/project/DefaultProject.java

```diff
@@ -137,7 +138,10 @@
 import static org.gradle.util.GUtil.addMaps;
 
 @NoConventionMapping
-public class DefaultProject extends AbstractPluginAware implements ProjectInternal, DynamicObjectAware {
+public class DefaultProject extends AbstractPluginAware implements ProjectInternal, DynamicObjectAware,
```

Comment:
Do we still want to rename fields on Project, too?

Reply:
No, because users don't extend `DefaultProject`, and the Groovy bug only affects subclassing.